src/org/sosy_lab/java_smt/solvers/princess/PrincessEnvironment.java

```diff
@@ -508,32 +540,69 @@ private static String getName(IExpression var) {
   }
 
   static FormulaType<?> getFormulaType(IExpression pFormula) {
+    // TODO: We could use Sort.sortof() here, but it sometimes returns `integer` even though the
```

Comment:
We should extract this into an issue so that we don't forget.

Reply:
I've opened a new Princess issue for this: https://github.com/uuverifiers/princess/issues/16